lib/bootloader_pvm.pm

```diff
@@ -42,30 +42,30 @@ sub get_into_net_boot {
     assert_screen 'pvm-bootmenu';
 
     # 5.   Select Boot Options
-    type_string "5\n";
+    enter_cmd "5";
```

Comment:
Hmm, we don't enter command, it really just types strings in the lpar boot menu. I find it confusing for this case.

Reply:
yes, but to confirm the *command* we still need to enter a newline, right? So same as in other cases we have the following choices:

1. `type_string "foo\n"`
2. `enter_cmd "foo"`
3. `type_string "foo"` and `send_key 'ret'`

which would you prefer?